Zilon.Core/Zilon.Emulation.Common/AutoplayEngineBase.cs

```diff
@@ -0,0 +1,271 @@
+
+                humanPlayer.MainPerson = person;
+
+                // TODO Использовать генератор персонажа, как в игре.
+                // Для этого нужно научить ботов корректно использовать оружие дальнего боя и посохи лечения.
+                var classRoll = new Random().Next(1, 3);
+                switch (classRoll)
+                {
+                    case 1:
+                        AddEquipmentToActor(person.EquipmentCarrier, 2, "short-sword", schemeService, propFactory);
+                        AddEquipmentToActor(person.EquipmentCarrier, 1, "steel-armor", schemeService, propFactory);
+                        AddEquipmentToActor(person.EquipmentCarrier, 3, "wooden-shield", schemeService, propFactory);
+                        break;
+
+                    case 2:
+                        AddEquipmentToActor(person.EquipmentCarrier, 2, "battle-axe", schemeService, propFactory);
+                        AddEquipmentToActor(person.EquipmentCarrier, 3, "battle-axe", schemeService, propFactory);
+                        AddEquipmentToActor(person.EquipmentCarrier, 0, "highlander-helmet", schemeService, propFactory);
+                        break;
+
+                    case 3:
+                        AddEquipmentToActor(person.EquipmentCarrier, 2, "bow", schemeService, propFactory);
+                        AddEquipmentToActor(person.EquipmentCarrier, 1, "leather-jacket", schemeService, propFactory);
+                        AddEquipmentToActor(inventory, "short-sword", schemeService, propFactory);
+                        AddResourceToActor(inventory, "arrow", 10, schemeService, propFactory);
+                        break;
+
+                    case 4:
+                        AddEquipmentToActor(person.EquipmentCarrier, 2, "fireball-staff", schemeService, propFactory);
+                        AddEquipmentToActor(person.EquipmentCarrier, 1, "scholar-robe", schemeService, propFactory);
+                        AddEquipmentToActor(person.EquipmentCarrier, 0, "wizard-hat", schemeService, propFactory);
+                        AddResourceToActor(inventory, "mana", 15, schemeService, propFactory);
+                        break;
+
+                    case 5:
+                        AddEquipmentToActor(person.EquipmentCarrier, 2, "pistol", schemeService, propFactory);
+                        AddEquipmentToActor(person.EquipmentCarrier, 0, "elder-hat", schemeService, propFactory);
+                        AddResourceToActor(inventory, "bullet-45", 5, schemeService, propFactory);
+
+                        AddResourceToActor(inventory, "packed-food", 1, schemeService, propFactory);
+                        AddResourceToActor(inventory, "water-bottle", 1, schemeService, propFactory);
+                        AddResourceToActor(inventory, "med-kit", 1, schemeService, propFactory);
+
+                        AddResourceToActor(inventory, "mana", 5, schemeService, propFactory);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this empty statement.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=5028977)